platforms/js/build_js.py

```diff
@@ -221,11 +226,24 @@ def build_doc(self):
     parser.add_argument('--skip_config', action="store_true", help="Skip cmake config")
     parser.add_argument('--config_only', action="store_true", help="Only do cmake config")
     parser.add_argument('--enable_exception', action="store_true", help="Enable exception handling")
+    parser.add_argument('--cmake_option', action='append', help="Append CMake options")
+    parser.add_argument('--build_flags',action='append',help="Append Emscripten build options")
     parser.add_argument('--build_wasm_intrin_test', default=False, action="store_true", help="Build WASM intrin tests")
+    parser.add_argument('--config', help="Specify configuration file with own list of exported into JS functions")
+#   Example of opencv_js.config.py file with config flag. We describe the functions and classes of the modul
+#     imgproc = {'': ['Canny']
+#               }
+#     core = {'': ["addWeighted"]
+#            }
+#
+#    white_list = makeWhiteList([core, imgproc])
```

Comment:
I think that the default config file will describe it better. So I'd like to recommend to remove this comment